lib/PuppeteerSharp/NetworkManager.cs

```diff
@@ -368,8 +363,8 @@ private void OnRequestWillBeSent(MessageEventArgs e)
             {
```

Comment:
not related to this pr but...
maybe change the if to `if (enabled == _protocolRequestInterceptionEnabled) return;` to match puppeteer's behavior?